doc/index.md

```diff
@@ -16,8 +16,12 @@ Welcome to the documentation for Deep Log Inspection
 * Installation & Administration Manual
     * Deep Log Inspection
         1. [Introduction](install/introduction.md)
-        2. [Installing & Deploying](install/install.md)
-        3. [Configuring the Monasca Log API](install/monasca-log-api.md)
-        4. [Configuring Kafka](install/kafka.md)
+        2. [Requirements](install/requirements.md)
+        3. [Installing & Deploying](install/install.md)
+        4. [Configuring Docker](install/docker.md)
+        5. [Configuring Traefik](install/traefik.md)
+        6. [Configuring the Monasca Log API](install/monasca-log-api.md)
+        7. [Configuring Kafka](install/kafka.md)
+        8. [Configuring the ELK stack](install/elk.md)
```

Comment:
just a quick tip (no need to change), with longer lists (say more than 3) it's better to use always `1.` The reason being convenience when you need to edit the list.